modules/virt-hot-plugging-disk-cli.adoc

```diff
@@ -1,26 +1,37 @@
 // Module included in the following assemblies:
 //
-// * virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+// * virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
 
 :_content-type: PROCEDURE
-[id="virt-hot-plugging-a-virtual-disk-using-the-cli_{context}"]
-= Hot plugging a virtual disk using the CLI
+[id="virt-hot-plugging-disk-cli_{context}"]
+= Hot plugging and hot unplugging a disk by using the command line
```

Comment:
In theory, hot plug and hot unplug should be separate modules, as they are different use cases.

Reply:
I see you deleted the hot unplug modules. I wonder if you should bring them back. Or, maybe have a single hot plug module with web console and cli instructions, and the same for a single hot unplug module.